library/spdm_common_lib/libspdm_com_context_data.c

```diff
@@ -688,6 +688,18 @@ libspdm_return_t libspdm_set_data(void *spdm_context, libspdm_data_type_t data_t
             context->max_spdm_session_sequence_number = LIBSPDM_MAX_SPDM_SESSION_SEQUENCE_NUMBER;
         }
         break;
+    case LIBSPDM_DATA_SPDM_VERSION_10_11_VERIFY_SIGNATURE_ENDIAN:
+        if (data_size != sizeof(uint32_t)) {
```

Comment:
I'm not understanding why this should be int. Should it not follow the flags (or endian) parameter in spdm_crypt_lib.h? (It's not an enum).

Reply:
Ah ok. There's a header issue then. For getting and setting the context `LIBSPDM_SPDM_10_11_VERIFY_SIGNATURE_ENDIAN_*` should be `#define`d in `spdm_common_lib.h`.

Reply:
Still not understanding. This data member of the context will be used as the "endian" in/out parameter to the verify function described by the previous comment where you were requesting it change from **uint32_t*** to **uin8_t***.  It will have one of these values:
```
/* The Endian values apply only if the spdm version is 1.0/1.1.
 * The default verification mode is big endian only. */
#define LIBSPDM_SPDM_10_11_VERIFY_SIGNATURE_ENDIAN_BIG_ONLY        (0x0)
#define LIBSPDM_SPDM_10_11_VERIFY_SIGNATURE_ENDIAN_LITTLE_ONLY     (0x1)
#define LIBSPDM_SPDM_10_11_VERIFY_SIGNATURE_ENDIAN_BIG_OR_LITTLE   (0x2)
```
And the parameter type to the verify function will be **uint8_t*** as you requested in the previous comment. 

So shouldn't think member in the context also be of type **uint8_t**?

Reply:
>So shouldn't think member in the context also be of type uint8_t?

Yes, but now the issue is that those `#define`s are currently in `spdm_crypt_lib.h`. They should be in `spdm_common_lib.h`.

Reply:
ok. Thanks.